plugins/module_utils/falconctl_utils.py

```diff
@@ -38,16 +39,24 @@
 # along with options.
 # Fake instantiation to make use of AnsibleModule funcs()
 _cs_path = "/opt/CrowdStrike"
-_FALCONCTL = get_bin_path(
-    'falconctl', required=True, opt_dirs=[_cs_path]
-)
+try:
+    _FALCONCTL = get_bin_path(
+        'falconctl', required=True, opt_dirs=[_cs_path]
+    )
+except ValueError:
+    FALCONCTL_NOT_FOUND = True
+    FALCONCTL_VALUE_ERROR = traceback.format_exc()
+else:
+    FALCONCTL_NOT_FOUND = False
```

Comment:
You could probably remove the else and make the `FALCONCTL_NOT_FOUND` default to `False`

Reply:
I basically just followed the ansible guide for this:
https://docs.ansible.com/ansible/latest/dev_guide/testing/sanity/import.html

Reply:
The big thing in this PR is fixing the ignore files - with these changes, we simply have to ignore the compile and import errors for 2.6 since we don't support that anyways.